src/browser/suggestion-box.js

```diff
@@ -202,7 +204,7 @@ define((require, exports, module) => {
       }, Icon[type] || ''),
       html.span({
         key: 'suggestion'
-      }, state.title)
+      }, type == 'history' ? state.uri : state.title)
```

Comment:
We should update this to match the spec, which displays `{title} — {domain}` for history entries.